spec/share/asciidoctor.spec.js

```diff
@@ -357,6 +357,25 @@ var shareSpec = function (testOptions, asciidoctor) {
         expect(paragraphBlock.getAttribute('bold-statement')).toBe('off');
       });
 
+      it('should get positional attributes by their 1-based index (getAttribute)', function () {
+        var doc = asciidoctor.load('[positional1,positional2,attr=value]\ntext');
+        var paragraphBlock = doc.getBlocks()[0];
+        expect(paragraphBlock.getAttribute(1)).toBe('positional1');
```

Comment:
I like the API, it's really simple to use.

Reply:
I still think it should be getPositionalAttribute(1) and getPositionalAttributes(). The positional attributes are intentionally supposed to be hidden. They are stored inside the attributes property as numbers merely as a convenience for the parser. The public API shouldn't see them there and should thus split them. That may then persuade me to change how I store them inside the parser.

Reply:
When I set up this system, i wasn't really thinking about the public API. That's why it seems so strange that a hash has both numeric and string keys. Given I had control over how the attributes were accessed, it was fine. But now it's seeping out.